test/e2e/gce/gce_test.go

```diff
@@ -0,0 +1,78 @@
+package gce
```

Comment:
can we move this to e2e? skip this test (t.Skip) if env PLATFORM is not GCE? i do not want to create a new pkg for this.